src/screens/status/status.vue

```diff
@@ -48,12 +48,12 @@
 import { mapState } from 'vuex'
 import pollAction from '@/mixins/pollAction'
 export default {
-  mixins: [pollAction('fetchNodeStatus')],
+  mixins: [pollAction('getNodeStatus')],
   data: () => ({
     node: process.env.AETERNITY_EPOCH_API_URL
   }),
   computed: mapState({
-    s: 'nodeStatus'
+    s: '$nodeStatus'
```

Comment:
Have you explain what `$` means somewhere?

Reply:
This is the syntax for the store:

- `${name of the variable}` -> global property unto the state
- `_{name of the module}` -> partials/component state, ex: modal, tooltip, etc..
- `{everything else}` -> modules

Reduces name collisions, makes it more clear where it comes from, though need to write this somewhere/readme.md

Reply:
Yes, if you introduce conventions, please document them.